cider/exec/nextgen/CMakeLists.txt

```diff
@@ -25,4 +25,9 @@ if(ENABLE_ASN)
   list(APPEND NEXTGEN_DEPS -fsanitize=address)
 endif()
 
+option(ENABLE_JITLIB_DEBUG "Enable debugging symbols for the JIT" OFF)
```

Comment:
Rethink this a little bit, why not make it config instead of build option? Then, we don't need to release a binary with optimizer turn on.

BTW, should be ```ENABLE_LLVM_OPTIMIZER```?

Reply:
Yes, I got you, have changed it to a config option